pack/parallel/ltr.json

```diff
@@ -0,0 +1,258 @@
+            "level": { "min": 0, "max": 3 }
+        },
+        {
+            "faction": ["neutral"],
+            "level": { "min": 0, "max": 5 }
+        },
+        {
+          "trait": ["spell", "cursed"],
+          "level": { "min": 0, "max": 4 }
+        },
+        {
+            "faction": ["survivor"],
+            "level": { "min": 0, "max": 0 },
+            "limit": 5,
+            "error": "You cannot have more than 5 Survivor cards."
+        }
+    ],
+    "deck_requirements": "size:30, card:02012:90050, card:02013:90051, random:subtype:basicweakness",
+    "double_sided": true,
+    "faction_code": "mystic",
+    "flavor": "“I know I can make the dead dance.”",
+    "health": 7,
+    "hidden": true,
+    "illustrator": "Magali Villeneuve",
+    "is_unique": true,
+    "name": "Jim Culver",
+    "pack_code": "ltr",
+    "position": 49,
+    "quantity": 1,
+    "sanity": 8,
+    "skill_agility": 2,
+    "skill_intellect": 3,
+    "skill_combat": 3,
+    "skill_willpower": 4,
+    "subname": "The Musician",
+    "text": "[reaction] After you reveal 1 or more [skull] or [curse] tokens during a skill test you are performing: Replenish 1 charge on an asset you control. (Limit once per round.)\n[elder_sign] effect: +2. You may treat this token as if it were a [skull] or [curse] token instead.",
+    "traits": "Performer. Cursed.",
+    "type_code": "investigator"
+  },
+  {
+    "code": "90050",
+    "cost": 2,
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "illustrator": "John Pacer",
+    "is_unique": true,
+    "name": "Jim's Trumpet",
+    "pack_code": "ltr",
+    "position": 50,
+    "quantity": 1,
+    "restrictions": "investigator:02004",
+    "skill_wild": 2,
+    "skill_willpower": 2,
+    "slot": "Hand",
+    "subname": "The Dead Speak",
+    "text": "Jim Culver deck only. Advanced.\n[reaction] When a [skull] or [curse] token is revealed during a skill test you are performing, exhaust Jim's Trumpet: Heal 1 horror from each investigator and [[Ally]] asset at your location or a connecting location.",
+    "traits": "Item. Instrument. Relic.",
+    "type_code": "asset"
+},
+{
+    "code": "90051",
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "flavor": "When the dead start talking about what scares them, that's when it's time to listen.",
+    "illustrator": "Preston Stone",
+    "name": "Final Rhapsody",
+    "pack_code": "ltr",
+    "position": 51,
+    "quantity": 1,
+    "restrictions": "investigator:02004",
+    "subtype_code": "weakness",
+    "text": "Advanced.\n<b>Revelation</b> - Reveal 5 chaos tokens from the chaos bag. For each non-[elder_sign], non-[bless] symbol token revealed, take 1 damage and 1 horror.",
+    "traits": "Endtimes.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "90052",
+    "cost": null,
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "illustrator": "John Pacer",
+    "is_unique": true,
+    "name": "Jim's Trumpet",
```

Comment:
Hello. I found the name of The Beyond (front side) is written as Jim's Trumpet, during translation.